uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-request-record-actions.scss

```diff
@@ -0,0 +1,33 @@
+leave-request-record-actions {
+  display: block;
+
+  .fa,
```

Comment:
```scss
leave-request-record-actions .fa
```
this applies a custom margin to an icon within a button, but we already have a "button with icon" component (see my other comment) that takes care of it

Reply:
Done